webtau-feature-testing/examples/scenarios/graphql/queryAndMutation.groovy

```diff
@@ -19,7 +19,7 @@ package scenarios.graphql
 import static org.testingisdocumenting.webtau.WebTauGroovyDsl.*
 
 def listAllQuery = '''
-query {
+{
```

Comment:
I didn't actually need to do this, was left over from debugging, but it's correct either way.